packages/frontier-libraries/stdlib/docs/.nojekyll

```diff
@@ -0,0 +1 @@
+TypeDoc added this file to prevent GitHub Pages from using Jekyll. You can turn off this behavior by setting the `githubPages` option to false.
```

Comment:
drop the docs folder and typedoc related stuff. We can look into that on the other issue